src/keyboard.rs

```diff
@@ -4,24 +4,24 @@ use winit::event::{ElementState, KeyEvent};
 use winit::keyboard::{Key as LogicalKey, KeyCode, ModifiersState, NamedKey, PhysicalKey};
 
 /// Some shortcuts use Cmd on Mac and Control on other systems.
-#[cfg(macos)]
+#[cfg(target_os = "macos")]
```

Comment:
We use [cfg_aliases](https://docs.rs/cfg_aliases/latest/cfg_aliases/) instead. Is it possible to tell clippy to remove this suggestion?

Reply:
Yes, it is! You can tell clippy inside _Cargo.toml_ to allow specific lints that would be denied by default.